dangerfile.js

```diff
@@ -12,6 +12,9 @@ let namesRemoved = []
 
 const isGame = game => /^games\/\w+\.yaml$/.test(game)
 
+let unknownLanguageDetected = false
+const knownLanguages = ['ActionScript', 'Ada', 'AngelScript', 'Assembly', 'Blitz BASIC', 'C', 'C#', 'C++', 'CoffeeScript', 'D', 'Delphi', 'Elm', 'F#', 'GDScript', 'GML', 'Go', 'Haskell', 'Haxe', 'Java', 'JavaScript', 'Kotlin', 'Lisp', 'Lua', 'Object Pascal', 'Objective-C', 'ooc', 'Pascal', 'Perl', 'PHP', 'Python', 'QBasic', 'QuakeC', 'QuickBASIC', 'Ruby', 'Rust', 'Scala', 'Swift', 'TorqueScript', 'TypeScript', 'Visual FoxPro']
```

Comment:
please expand this so that every entry is in one line

Reply:
is expanded